File: c++/HelloX.cpp
#include "stdafx.h"
#include <iostream>
#include <string>

using namespace std;

class HelloX {

public:
	static void helloX()
	{
		cout << "What is your name?" << endl;

		string name;
		getline(cin, name);

		cout << "Hello " << name << endl;
	};
};

File: java/HelloX.java
package io.github.moppletop.quickproblems;

import java.util.Scanner;

public class HelloX {

	public static void helloX() {
		System.out.println("What is your name?");

		Scanner scanner = new Scanner(System.in);
		String name = scanner.nextLine();

		System.out.println("Hello " + name);
		scanner.close();
	}

}
